### DISCUSSION TITLE
Can't delete row

### DISCUSSION THREAD

**Discussion by cyber-kivi** [Budibase/budibase#6201] [Help]
Hello there!

I have a problem with deleting selected row. 

Main table with link 
![image](https://user-images.githubusercontent.com/106427850/172233241-4745b016-6ca3-42f9-801e-5831afe57c61.png)

Edit screen
![image](https://user-images.githubusercontent.com/106427850/172233333-7a7ad7d3-a84d-41bb-86f7-2a746abcd919.png)

Filtering
![image](https://user-images.githubusercontent.com/106427850/172233616-9cf0393b-c94b-4bab-985d-f39fc41e7e46.png)

1.Select row
![image](https://user-images.githubusercontent.com/106427850/172234440-ab0e1b2c-2e78-40eb-b751-d163c29ea0df.png)
2. Delete row
![image](https://user-images.githubusercontent.com/106427850/172233886-50116f02-bbea-4263-9cc2-1af966e466d2.png)
3. Confirm delete
![image](https://user-images.githubusercontent.com/106427850/172233941-2148640c-bd64-479e-a78d-c9ff2a96d832.png)
4. Selected row still in main table
![image](https://user-images.githubusercontent.com/106427850/172233990-8a3ebc12-48ba-4ba4-8514-4e895fddf3b5.png)

**Comment by melohagan** [comment]
Hi @cyber-kivi 

Looks okay to me from the screenshots. Would you be able to share your app export to support@budibase.com

May also be worth updating your app - the blue update cycle shows there's an update available.
